.github/quarkus-github-bot.yml

```diff
@@ -406,6 +412,9 @@ triage:
     - id: mandrel
       labels: [area/mandrel]
       titleBody: "mandrel"
+      expression: |
+        matches("mandrel", titleBody)
+        || (matches("native", titleBody) && !matches("knative", titleBody))
```

Comment:
is "native" not going to add alot of "noise" to the mandrel category?

Reply:
this is the one I debated about, problem is lot's of discussions mention native and have a problem that only happens in native mode, do you have another category suggestion?

Reply:
The `area/native-image` label exists for the purpose of categorizing issues related to the native generation (regardless if using Mandrel or GraalVM CE)

Reply:
ok I'll move it to that instead this evening.